src/modules/rover_ackermann/module.yaml

```diff
@@ -6,26 +6,37 @@ parameters:
       RA_WHEEL_BASE:
         description:
           short: Wheel base
-          long: Distance from the front to the rear axle
+          long: Distance from the front to the rear axle.
         type: float
         unit: m
-        min: 0.001
+        min: 0
         max: 100
         increment: 0.001
         decimal: 3
-        default: 0.5
+        default: 0
```

Comment:
Doesn't have to be in this PR, but it would improve the UX if you'd publish a warning if one of the params isn't configured, instead of failing silently.
As inspiration, [this is what we have for FW.](https://github.com/PX4/PX4-Autopilot/blob/e7e76e2e2124239319768e1f3e4e7b8942797bcf/src/lib/fw_performance_model/PerformanceModel.cpp#L164)